backend/src/database/models/user.ts

```diff
@@ -1,17 +1,34 @@
-import * as mongoose from 'mongoose'
+import * as mongoose from 'mongoose';
 
 const userSchema = new mongoose.Schema({
     name: {
         type: String,
-        required: true,
+        required: false
```

Comment:
nie powinniśmy wymagać name? Coś trzeba będzie wyświetlić w GUI jako nazwę użytkownika

Reply:
czy w endpoincie do rejestrowania powinien być dodatkowy parametr **name**? ja na razie myślałem że na samym początku będzie /register: tylko rejestracja (mail + hasło) a pozostałe pola dotyczące usera będą podawane w jakimś następnym endpoincie /create-user czy tam /update-user

Reply:
ok, to jest narazie minor. Zmieni się to później jak będziemy mieć potrzebę